bsdlib/doc/CHANGELOG.rst

```diff
@@ -5,6 +5,13 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.6.0
+************
+
+* Removed ``nrf_inbuilt_key`` API. How to handle keys will from now on be up to the application.
+* Removed ``nrf_apn_class`` API. How to handle APN Class will from now on be up to the application.
+* Removed crypto dependancy towards ``nrf_oberon`` from the library. The Library does not need any special cryptography functions anymore as signing of AT commands will now be handled by the application.
+
```

Comment:
``` 
* Removed the ``nrf_inbuilt_key`` API. 
  From now on, the application is responsible for handling keys.
* Removed the ``nrf_apn_class`` API. 
  From now on, the application is responsible for handling the Access Point Name (APN) class.
* Removed the crypto dependency towards ``nrf_oberon`` from the library. 
  The library does not need any special cryptography functions anymore, because the application is now responsible for signing AT commands.
```

Reply:
@ru-fu I have updated the changelog entries as you have rephrased them.